src/main/java/org/jpeek/calculus/java/Ccm.java

```diff
@@ -105,4 +122,92 @@ private static void updateNcc(
             ).toString()
         );
     }
+
+    /**
+     * Adds a meta tag to the skeleton XML.
+     * @param skeleton The skeleton XML to which the meta tag will be added
+     * @return The modified XML with the meta tag added
+     */
+    private static XML addMetaTag(final XML skeleton) {
+        final DocumentBuilderFactory factory = DocumentBuilderFactory.newInstance();
+        DocumentBuilder builder = null;
+        final Document doc;
+        try {
+            builder = factory.newDocumentBuilder();
+            doc = builder.newDocument();
+            final Element meta = doc.createElement("meta");
+            Node packages = skeleton.node().getFirstChild().getFirstChild().getNextSibling()
```

Comment:
Really near to impossible to support this code, I can not remember the order of the nodes in XML and nobody wants, that's why there are a lot of ways to work with XML using XSL, XPATH, tag names. 
If you walk with order only, your code will be broken by any sceleton structure changes.